lib/components/map/route-preview-overlay.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { connect } from 'react-redux'
+import { FeatureGroup, MapLayer, Polyline, withLeaflet } from 'react-leaflet'
+import polyline from '@mapbox/polyline'
+import React from 'react'
+
+import {
+  getActiveItinerary,
+  getActiveSearch,
+  getVisibleItineraryIndex
+} from '../../util/state'
+
+/**
+ * This overlay will display thin gray lines for a set of geometries. It's to be used
+ * as a stopgap until we make full use of Transitive!
+ */
+class GeometryPreviewOverlay extends MapLayer {
```

Comment:
I was going to say it would be better to show nothing when you hover on the background instead of grey lines. But leave in if it's a placeholder for a later addition

Reply:
This was *hotly* debated lol. We settled on gray lines because every trip planner shows something on the map once the query has finished loading. It looked weird with nothing coming up on page load.

Reply:
This will have to be changed during the move to vector tiles anyway, so it's quite temporary for now